### PR TITLE
Front end views, basic routing and stuff.

### PR DESCRIPTION
**Pull request by adriansdk**

### CODE REVIEW

File: routes/beef.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    res.render('beef');
+});
+
+// router.get('/api-key', (req, res, next) => {
+//     res.json({
+//         GOOGLEMAPS_API_KEY: process.env.GOOGLEMAPS_API_KEY
+//     })
+// })
+
+module.exports = router;
```
Comment (AndresMWeber) [inline routes/beef.js]: Should add a new line after this

File: routes/login.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    res.render('./login-views/login');
+});
+
+// router.get('/api-key', (req, res, next) => {
+//     res.json({
+//         GOOGLEMAPS_API_KEY: process.env.GOOGLEMAPS_API_KEY
+//     })
+// })
+
+module.exports = router;
```
Comment (AndresMWeber) [inline routes/login.js]: New line again

File: routes/signup.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    res.render('./signup-views/signup');
+});
+
+// router.get('/api-key', (req, res, next) => {
+//     res.json({
+//         GOOGLEMAPS_API_KEY: process.env.GOOGLEMAPS_API_KEY
+//     })
+// })
+
+module.exports = router;
```
Comment (AndresMWeber) [inline routes/signup.js]: new line!

File: views/beef.hbs [hunk 4]
```diff
@@ -0,0 +1 @@
+<H1>THIS IS BEEF</H1>
```
Comment (AndresMWeber) [inline views/beef.hbs]: new liiiiiiiine

File: views/login-views/login.hbs [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+<div class="login">
+  <form action="/login" method="POST">
+    <p>Name:</p>
+    <input type="text" name="username" placeholder="Your name">
+    <p>Password:</p>
+    <input type="password" name="password" placeholder="Your password">
+    <button class="btn btn-danger"> Login </button>
+  </form>
+</div>
```
Comment (AndresMWeber) [inline views/login-views/login.hbs]: new line

File: views/signup-views/signup.hbs [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  <form action="/signup" method="POST">
+    <p>Name:</p>
+    <input type="text" name="username" placeholder="Your name">
+    <p>Password:</p>
+    <input type="password" name="password" placeholder="Your password">
+    <button class="btn btn-danger"> Create Account</button>
+    <p>Email:</p>
+    <input type="email" name="email" placeholder="...@gmail.com">
+  </form>
+</div>
```
Comment (AndresMWeber) [inline views/signup-views/signup.hbs]: new line

**Comment by adriansdk** [comment]
After adjusting some minor requests, managed to merge without conflicts.

**Pull request merged**